inc/healthcheck/namespace.php

```diff
@@ -26,35 +26,75 @@ function bootstrap() {
 		return;
 	}
 
+	// Defer app level healthchecks until after plugins have laoded.
+	add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_healthcheck', 100 );
+
+	// Run instance healthcheck immediately.
+	if ( strpos( $_SERVER['REQUEST_URI'], '/__instance_healthcheck' ) !== false ) {
```

Comment:
Oh, maybe this should have been `=== 0` btw @roborourke?

Reply:
I guess was just same as before

Reply:
Mayyybe. Could it ever be the case that Altis is served from a subdirectory? I know we don't really support that. Seems like something that's not really gonna present a problem given the double underscore prefix.

Reply:
> Could it ever be the case that Altis is served from a subdirectory?

No, it'll crash the server iirc if you do (including with the existing healthcheck).

> Seems like something that's not really gonna present a problem given the double underscore prefix.

Yeah, just thinking of what the "best" is